src/data/projects.js

```diff
@@ -208,6 +208,14 @@ const projects = [
     website: "https://satnogs.org/",
     image: "SatNOGS.png"
   },
+  {
+    name: "NEODyS-2",
+    dateAdded: "31/08/2023",
+    tags: ["Near Earth Objects", "Asteroids","OSINT"],
+    description: "NEODyS provides information and services for all Near-Earth Asteroids (NEAs). ",
```

Comment:
I don't know if the space before the full stop has any impact on the deploy, but I guess not.